### DISCUSSION TITLE
NameError for playerDraw variable

### DISCUSSION THREAD

**Discussion by SujalSamai** [CSCult/uno-card-game#6] [Q&A] [answered]
![error](https://user-images.githubusercontent.com/87236576/127495242-45d2a862-4aae-41f5-8b31-b2fe400e462b.png)
While all the functions and logics are working properly, the game whenever reaches to the cards "Draw Two" or "Draw Four" it gives a NameError for playerdraw variable present at line no. 152. Here is a screenshot attached of the same. Please tell a solution if available.

**Comment by UditThakkar** [comment]
Let me check

**Comment by UditThakkar** [comment, marked as answer]
So in line 160 and 167 where you appended the extra card you have to append them in the players list not for plyerTurn like this:
players[playerTurn].extend....
we have to add those cards in the card pile for like 1st player.

**Reply by SujalSamai** [reply]
Thank You
